src/modules/sensors/vehicle_imu/VehicleIMU.cpp

```diff
@@ -116,17 +113,19 @@ bool VehicleIMU::ParametersUpdate(bool force)
 		parameter_update_s param_update;
 		_parameter_update_sub.copy(&param_update);
 
-		const auto imu_integ_rate_prev = _param_imu_integ_rate.get();
-
 		updateParams();
 
 		updated = true;
 
-		const auto accel_calibration_count = _accel_calibration.calibration_count();
-		const auto gyro_calibration_count = _gyro_calibration.calibration_count();
 		_accel_calibration.ParametersUpdate();
 		_gyro_calibration.ParametersUpdate();
 
+		const auto accel_calibration_count = _accel_calibration.calibration_count();
```

Comment:
Why did this move? It's need to grab the calibration count before the accel calibration parameters are updated, then we know below if it actually changed.

Reply:
true, seems I was a bit overeager and didn't take enough time to understand what's going on.

Reverted it.